### PR TITLE
Adds Enterprise Database Chart.

### PR DESCRIPTION
**Pull request by rodrigovalin**
# Summary

Adds a simple `enterprise-database` Chart.

### CODE REVIEW

File: charts/enterprise-database/README.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+helm install <resource-name> mongodb/enterprise-database --namespace mongodb [--create-namespace]
+```
+
+To install the Enterprise Operator in a namespace called `mongodb`; with the
+optional `--create-namespace` Helm will create the Namespace if it does not exist.
+
+## Configuring access to Cloud Manager
+
+You need your Ops Manager or Cloud Manager stored in a `Secret`, please refer to
+the [MongoDB Enterprise Kubernetes Operator](../enterprise-operator) docs about how
```
Comment (jasonmimick) [inline charts/enterprise-database/README.md:42]: does this link work?

File: charts/enterprise-database/README.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+to create them. Additionally, a `ConfigMap` will need to be created pointing at your
+Cloud Manager or Ops Manager installation.
```
Comment (jasonmimick) [inline charts/enterprise-database/README.md:44]: Can we add some simple little script that can use mongocli, or even link to the docs which have the example ConfigMap and Secret? e.g. https://github.com/mongodb-developer/get-started-aws-cfn/blob/main/export-mongocli-config.py

File: charts/enterprise-database/templates/database.yaml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+---
+apiVersion: mongodb.com/v1
+kind: MongoDB
+metadata:
+  name: {{ .Release.Name }}
+  namespace: {{ .Release.Namespace }}
+spec:
+  type: {{ .Values.mongodb.type }}
+  members: {{ .Values.mongodb.members | default 3 }}
+  version: {{ .Values.mongodb.version | default "5.0.4" }}
```
Comment (jasonmimick) [inline charts/enterprise-database/templates/database.yaml:12]: How can we maintain this default version?

**Review by jasonmimick — APPROVED** [review]
Looks really great - left a couple comments on what we can do to make it even easier.

File: charts/enterprise-database/README.md (hunk 1 above)
Comment (rodrigovalin) [reply charts/enterprise-database/README.md:42]: Yes, the same as [this one](https://github.com/mongodb/helm-charts/tree/main/charts/community-operator#prerequisites) (from community-operator to community-operator-crds

File: charts/enterprise-database/templates/database.yaml (hunk 3 above)
Comment (rodrigovalin) [reply charts/enterprise-database/templates/database.yaml:12]: Maintain as in bumping this version whenever a new Mongodb version is released? That will have to be done manually for now.

File: charts/enterprise-database/README.md (hunk 2 above)
Comment (rodrigovalin) [reply charts/enterprise-database/README.md:44]: I'm linking to the `enterprise-operator` Chart, which in turns points at [the official docs](https://docs.cloudmanager.mongodb.com/tutorial/nav/k8s-config-for-mdb-resource/). Do you think that's enough? maybe we can discuss a better approach for the future

File: charts/enterprise-database/templates/database.yaml (hunk 3 above)
Comment (jasonmimick) [reply charts/enterprise-database/templates/database.yaml:12]: ok thanks

**Pull request merged**